### PR TITLE
fix: set output from fmt to log for server run in daemon

### LINKED ISSUE #991: when service run in daemon way, log printed by fmt is lost
**Describe the bug**
When run daemon, can't find below message in any log file

`fmt.Printf("heartbeatPort[%v],replicaPort[%v]\n",` m.config.heartbeatPort, `m.config.replicaPort)`

**Environment**
Describe your server system version, kernel version

**To Reproduce**

**Expected behavior**

**Additional context**

### PR DESCRIPTION
**Pull request by Victor1319**
Signed-off-by: Victor1319 <834863182@qq.com>

**What this PR does / why we need it**:
modify the way to print log info from fmt to log in daemon server

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #991

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: master/topology.go [hunk 1]
```diff
@@ -349,7 +350,7 @@ func (t *topology) allocZonesForMetaNode(zoneNum, replicaNum int, excludeZone []
 
 func (t *topology) allocZonesForDataNode(zoneNum, replicaNum int, excludeZone []string) (zones []*Zone, err error) {
 	zones = t.getAllZones()
-	fmt.Printf("len(zones) = %v \n", len(zones))
+	syslog.Printf("len(zones) = %v \n", len(zones))
```
Comment (zhuhyc) [inline master/topology.go:353]: use log.LogInfof instead of syslog.Printf

File: master/topology.go [hunk 2]
```diff
@@ -629,7 +630,7 @@ func (zone *Zone) canWriteForDataNode(replicaNum uint8) (can bool) {
 		}
 		return true
 	})
-	fmt.Printf("canWriteForDataNode leastAlive[%v],replicaNum[%v],count[%v]\n", leastAlive, replicaNum, zone.dataNodeCount())
+	syslog.Printf("canWriteForDataNode leastAlive[%v],replicaNum[%v],count[%v]\n", leastAlive, replicaNum, zone.dataNodeCount())
```
Comment (zhuhyc) [inline master/topology.go:633]: use log.LogInfof instead of syslog.Printf

**Review by zhuhyc — DISMISSED** [review]

File: master/server.go [hunk 3]
```diff
@@ -163,7 +164,7 @@ func (m *Server) checkConfig(cfg *config.Config) (err error) {
 	if m.config.replicaPort <= 1024 {
 		m.config.replicaPort = raftstore.DefaultReplicaPort
 	}
-	fmt.Printf("heartbeatPort[%v],replicaPort[%v]\n", m.config.heartbeatPort, m.config.replicaPort)
+	syslog.Printf("heartbeatPort[%v],replicaPort[%v]\n", m.config.heartbeatPort, m.config.replicaPort)
```
Comment (mervinkid) [inline master/server.go:167]: Use `log.LogInfof` instead of `syslog.Printf`.

File: master/server.go [hunk 4]
```diff
@@ -250,7 +251,7 @@ func (m *Server) createRaftServer() (err error) {
 	if m.raftStore, err = raftstore.NewRaftStore(raftCfg); err != nil {
 		return errors.Trace(err, "NewRaftStore failed! id[%v] walPath[%v]", m.id, m.walDir)
 	}
-	fmt.Printf("peers[%v],tickInterval[%v],electionTick[%v]\n", m.config.peers, m.tickInterval, m.electionTick)
+	syslog.Printf("peers[%v],tickInterval[%v],electionTick[%v]\n", m.config.peers, m.tickInterval, m.electionTick)
```
Comment (mervinkid) [inline master/server.go:254]: Use `log.LogInfof` instead of `syslog.Printf`.

**Review by mervinkid — CHANGES_REQUESTED** [review]

**Review by mervinkid — APPROVED** [review]

**Comment by mervinkid** [comment]
Approved and will be merge after the CI tests finished.

**Pull request merged**